appengine-java8/tasks/README.md

```diff
@@ -69,12 +69,15 @@ location is "us-central1").
 export LOCATION_ID=<YOUR_ZONE>
 ```
 
+### Using App Engine Queues
 Create a task, targeted at the `/tasks/create` endpoint, with a payload specified:
 
 ```
 mvn exec:java -Dexec.mainClass="com.example.task.CreateTask" \
```

Comment:
Are these instructions still correct? It looks like you are using environment variables now

Reply:
This is correct. This readme is associated with the App Engine sample.